app/services/json_to_events.rb

```diff
@@ -19,16 +19,35 @@ def call
       campus = Campus.where(name: event_hash["campus"]).first_or_create!
 
       # Create an event
-      Event.create!(
+      # Event.create!(
+      #   campus: campus,
+      #   title: event_hash["title"],
+      #   description: event_hash["description"],
+      #   url: event_hash["url"],
+      #   foreign_id: event_hash["id"],
+      #   start_at: DateTime.parse(event_hash["startdt"]),
+      #   end_at: DateTime.parse(event_hash["enddt"]),
+      # )
+
+      Event.where(foreign_id: event_hash["id"]).first_or_create!(
         campus: campus,
         title: event_hash["title"],
         description: event_hash["description"],
         url: event_hash["url"],
         foreign_id: event_hash["id"],
         start_at: DateTime.parse(event_hash["startdt"]),
         end_at: DateTime.parse(event_hash["enddt"]),
-      )
+      ).update(
+        title: event_hash["title"],
+        description: event_hash["description"],
+        url: event_hash["url"],
+        start_at: DateTime.parse(event_hash["startdt"]),
+        end_at: DateTime.parse(event_hash["enddt"]),
+        )
+
     end
+    puts Event.last.start_at
```

Comment:
Do we need to leave this `puts` in here?

Reply:
No we don't